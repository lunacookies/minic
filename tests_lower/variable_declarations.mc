func main {
	var x = 10
	var y = 20
	var z = x
}
