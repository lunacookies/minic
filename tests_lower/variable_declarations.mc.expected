func main
	var x i64
	var y i64
	var z i64
	{
		set x = 10
		set y = 20
		set z = x
	}
